courses/ahybrid/v1.0/AHYBRID050/scripts/create_gke.sh

```diff
@@ -16,69 +16,65 @@
 
 source ./scripts/env.sh
 
-# create the gke cluster
 gcloud config set compute/zone ${C1_ZONE}
 gcloud beta container clusters create ${C1_NAME} \
     --machine-type=n1-standard-4 \
     --num-nodes=4 \
     --workload-pool=${WORKLOAD_POOL} \
     --enable-stackdriver-kubernetes \
     --subnetwork=default \
-    --labels=mesh_id=${MESH_ID} \
-    --release-channel=regular
```

Comment:
You added release-channel into all the labs, why remove it now?

Reply:
I removed it because I was isolating issues to get the build to work. I ended up replicating the 045 ASM install which didn't enable the release channel. The release channel would, among other things, force a newer version of K8s. I added an issue to the backlog to migrate 050 to release channel clusters in a future release.

Reply:
sgtm